lib/models/instance.js

```diff
@@ -349,6 +343,33 @@ Instance.prototype.getElasticHostname = function () {
   });
 };
 
+/**
+ * Get name for current branch
+ * @return {String}
+ */
+Instance.prototype.getBranchName = function () {
+  return keypather.get(this.contextVersion, 'getMainAppCodeVersion().attrs.branch');
+};
+
+/**
+ * Get name for current repository
+ * @return {String}
+ */
+Instance.prototype.getRepoName = function () {
+  var repo = keypather.get(this.contextVersion, 'getMainAppCodeVersion().attrs.repo');
+  if (repo) {
+    return repo.split('/')[1];
+  }
+};
+
+/**
+ * Get `repo/branch` style string for instance
+ *
+ * Function here for backwards compatibility. Use `getNameAndBranchName` instead
+ *
+ * @deprecated
```

Comment:
You're right. This shouldn't be deprecated since we're going to leave it there anyways. Just want to make sure it's not used improperly since now we'll have multiple repo containers.